### PR TITLE
Modeler v5 compatibility

### PR DESCRIPTION
**Pull request by rami-nk**
Fixed language selection dropdown according to https://camunda.com/blog/2022/01/how-to-migrate-your-camunda-modeler-plugins-to-work-with-camunda-modeler-5-x/.
Modal is still used instead of Overlay component.

During the migration missing translation keys were noticed.
Below is the list of the missing translations keys.

Missing translation: Compensation
Missing translation: Link
Missing translation: Multi-instance
Missing translation: Timer
Missing translation: History cleanup
Missing translation: Tasklist
Missing translation: Candidate starter
Missing translation: External task
Missing translation: Process variables
Missing translation: User assignment
Missing translation: Form fields
Missing translation: Task listeners
Missing translation: Start initiator
Missing translation: Called element
Missing translation: Asynchronous continuations
Missing translation: Job execution
Missing translation: In mapping propagation
Missing translation: In mappings
Missing translation: Inputs
Missing translation: Connector inputs
Missing translation: Out mapping propagation
Missing translation: Out mappings
Missing translation: Outputs
Missing translation: Connector outputs
Missing translation: Execution listeners
Missing translation: Extension properties
Missing translation: Field injections
Missing translation: Business key
Missing translation: Version tag
Missing translation: Element documentation
Missing translation: Time to live
Missing translation: Candidate starter groups
Missing translation: Candidate starter users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Camunda Modeler I18N-Plugin
+[![Compatible with Camunda Modeler version 5.0](https://img.shields.io/badge/Camunda%20Modeler-5.0+-blue.svg)](https://github.com/camunda/camunda-modeler)
 
 This plugin allows you to translate the UI of the Camunda Modeler. It contains translations for German, English and Portuguese and can be easily extended or customized.
```
Comment (alex-praschek) [inline README.md]: Chinese (Traditional), Chinese (Simplified), Dutch, and Russian are missing in the description. Could you update this please?

**Review by alex-praschek — APPROVED** [review]

**Comment by rami-nk** [comment]
I've updated README.md and added German translations for the missing keys.

**Review by alex-praschek — APPROVED** [review]
Please align with @dominikhorn93 about the translations marked as TODO. Thanks!

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 2]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'Java class': 'Java Klasse',
+    'Delegate expression': 'Delegierte Expression',
+    'Field injection': 'Feld-Injektionen',
+    'Errors': 'Fehler',
+    'Task listener': 'Task listener',
+    'Task listeners': 'Task listener',
+    'In mappings': 'In mappings',
+    'Set Color': 'In mappings',
+    '<none>': ' - ',
+    'Camunda Forms': 'Camunda Formen',
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: Camunda Formulare

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 3]
```diff
@@ -198,5 +198,65 @@ export default {
...
     '[unknown template: {templateId}]': '[Unbekanntes Template: {templateId}]',
-    '{label} must not contain whitespace': '{label} darf keine Leerzeichen enthalten'
+    '{label} must not contain whitespace': '{label} darf keine Leerzeichen enthalten',
+    'Compensation': 'Kompensatiion',
+    'Link': 'Link',
+    'Multi-instance': 'Multi-Instanz',
+    'Timer': 'Timer',
+    'History cleanup': 'Verlauf löschen',
+    'Tasklist': 'Taskliste',
+    'Candidate starter': 'Kandidatenstarter', // TODO
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: würde ich nicht übersetzen

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 4]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'External task': 'Exterer Task',
+    'Process variables': 'Prozessvariablen',
+    'User assignment': 'Benutzeraufgabe',
+    'Form fields': 'Formfelder',
+    'Start initiator': 'Start-Initiator',
+    'Called element': 'Aufgerufenes Element', // TODO
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: // TODO löschen

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 5]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'Asynchronous continuations': 'Asynchrone Fortsetzung', // TODO
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: // TODO löschen

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 6]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'Job execution': 'Jobausführung',
+    'In mapping propagation': 'In mapping propagation', // TODO
+    'In mapping': 'In mapping', // TODO
+    'Inputs': 'Eingaben',
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js:219]: Inputs würde ich nicht übersetzen

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 7]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'Connector inputs': 'Connector-Eingaben',
+    'Out mapping propagation': 'Out mapping propagation',
+    'Out mappings': 'Out mappings',
+    'Outputs': 'Ausgaben',
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: Outputs würde ich nicht übersetzen

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 8]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'Connector outputs': 'Connector-Ausgaben',
+    'Execution listeners': 'Ausführungs-Listener',
+    'Extension properties': 'Zusätzliche Eigenschaften', // TODO
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: //TODO entfernen

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 9]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'Change element': 'Element ändern',
+    'Inline script': 'Inline-Skript',
+    'Format': 'Format',
+    'External resource': 'Externe Ressource',
+    'Start typing "${}" to create an expression.': 'Beginnen Sie mit der Eingabe von "${}", um eine Expression zu erstellen.',
+    'Created in': 'Erstellt in',
+    'Process variable name': 'Prozess-Variablenname',
+    'Embedded or External Task Forms': 'Eingebettete oder externe Task-Formen',
+    'Generated Task Forms': 'Generierte Task-Formen',
+    'Form key': 'Form-Key',
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: Formular Key

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 10]
```diff
@@ -198,5 +198,65 @@ export default {
...
+    'Form reference': 'Form-Referenz'
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: Formular Referenz

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js [hunk 11]
```diff
@@ -198,5 +198,65 @@ export default {
...
-    '{label} must not contain whitespace': '{label} darf keine Leerzeichen enthalten'
+    '{label} must not contain whitespace': '{label} darf keine Leerzeichen enthalten',
+    'Compensation': 'Kompensatiion',
+    'Link': 'Link',
+    'Multi-instance': 'Multi-Instanz',
+    'Timer': 'Timer',
+    'History cleanup': 'Verlauf löschen',
+    'Tasklist': 'Taskliste',
+    'Candidate starter': 'Kandidatenstarter', // TODO
+    'External task': 'Exterer Task',
```
Comment (dominikhorn93) [inline client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: Externer

**Review by dominikhorn93 — CHANGES_REQUESTED** [review]
Danke dafür!
Habe ein paar Änderungen vorgeschlagen

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js (hunk 2 above)
Comment (alex-praschek) [reply client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: Würde das nicht übersetzen, ist ein Eigenname

File: client/bpmnjs-i18n-extension/languages/de/properties-panel.js (hunk 9 above)
Comment (alex-praschek) [reply client/bpmnjs-i18n-extension/languages/de/properties-panel.js]: Formular-Key

**Review by dominikhorn93 — APPROVED** [review]

**Pull request merged**